atest/robot/output/xunit.robot

```diff
@@ -89,3 +113,18 @@ Suite Stats Should Be
     Element Attribute Should Be       ${elem}    skipped     ${skipped}
     Element Attribute Should Match    ${elem}    time        ?.???
     Element Attribute Should Be       ${elem}    errors      0
+    Element Attribute Should Match    ${elem}    timestamp   ????-??-??T??:??:??.???000
+
+Verify XUnit Timestamp
+    [Arguments]    ${pattern}
+    File Should Exist    ${OUTDIR}/xunit.xml
+    ${suite} =    Get XUnit Node
+    Should match    ${suite.attrib['timestamp']}    ${pattern}
+
+Execute On Existing Execution Environment
+    [Arguments]    ${executor}    ${options}    ${sources}    ${default options}=
+    @{arguments} =    Get Execution Arguments    ${options}    ${sources}    ${default options}
+    ${result} =    Run Process    @{executor}    @{arguments}
+    ...    stdout=${STDOUTFILE}    stderr=${STDERRFILE}    output_encoding=SYSTEM
+    ...    timeout=5min    on_timeout=terminate
+    [Return]    ${result}
```

Comment:
I still don't understand the need for this special keyword. Why couldn't test just use `Run Rebot`?